version/version.go

```diff
@@ -6,12 +6,12 @@ import (
 
 var (
 	// Version is the main version number that is being run at the moment.
-	Version = "0.0.1"
+	Version = "0.25.1"
```

Comment:
- I was confused when I saw your PR title mentioning `0.25` while this version is instead apparently `0.25.1`. You might want to update your PR title, so that the merge commit that gets created once this lands is also correct and not misleading.
 - If I'm not mistaken, I saw that the GitHub Release + tag you created was pointing to a commit from this `release/0.25.1` branch while you were still working on it. I wonder if it would make more sense to move the tag after this PR gets merged, to point it to the new `main` (and thus also including all the changes you made—or will add make to address feedback). That being said, it's also not ideal to move the tag that has already been created (and potentially used). Alternatively, we could consider doing a `0.25.2` after this PR lands just to realign things (even if the only changes between the existing `0.25.1` tag and that future `0.25.2` would be changes around the release tooling, not the packer plugin itself)?

Reply:
Yeah that's the thing: the previously released version never made to `main`, so this PR is a first step to get things aligned.

I wanted to build a new release with the fix in order to test it ASAP, so I created the tag from the branch and then the new version as soon as I saw everything was green on CI. I guess `0.25.0` was done locally as my builds were the first ones in the pipeline.

I kept the title as `0.25` as a way to indicate I'm merging ALSO what was done on `0.25.0`, but yeah on a second thought that also doesn't make things clearer.

I'm preparing a new PR with some updates discussed here and was indeed planning on a `0.25.2` once that one is merged 👍 so I'd rather keep this one as is.

Reply:
> Yeah that's the thing: the previously released version never made to main, so this PR is a first step to get things aligned.
> 
> I wanted to build a new release with the fix in order to test it ASAP, so I created the tag from the branch and then the new version as soon as I saw everything was green on CI.

Ahah, that's exactly what I figured and guessed happened 😜   WFM 👍